shared/chat/inbox/row/avatars.js

```diff
@@ -19,37 +19,33 @@ function rowBorderColor(idx: number, isLastParticipant: boolean, backgroundColor
 
 type AvatarProps = {
   participants: Array<string>,
-  isHovered: boolean,
   isLocked: boolean,
   isMuted: boolean,
   isSelected: boolean,
   backgroundColor: string,
 }
 
-const MutedIcon = ({isHovered, isMuted, isSelected, isLocked}) => {
-  let icon = null
-  let type
-  if (isMuted) {
-    if (isMobile) {
-      type = isSelected ? 'icon-shh-active-24' : 'icon-shh-24'
-    } else {
-      type = isSelected ? 'icon-shh-active-16' : isHovered ? 'icon-shh-hover-16' : 'icon-shh-16'
-    }
-    icon = <Icon type={type} style={avatarIconStyle} />
-  } else if (isLocked) {
-    if (isMobile) {
-      type = isSelected ? 'icon-addon-lock-active-22' : 'icon-addon-lock-22'
-    } else {
-      type = isSelected
-        ? 'icon-addon-lock-active-16'
-        : isHovered
-          ? 'icon-addon-lock-hover-16'
-          : 'icon-addon-lock-16'
-    }
-    icon = <Icon type={type} style={avatarIconStyle} />
-  }
-  return icon
-}
+const MutedIconMobile = ({isLocked, isMuted, isSelected}) =>
+  isMuted ? (
+    <Icon type={isSelected ? 'icon-shh-active-24' : 'icon-shh-24'} style={avatarIconStyle} />
+  ) : isLocked ? (
+    <Icon type={isSelected ? 'icon-addon-lock-active-22' : 'icon-addon-lock-22'} style={avatarIconStyle} />
+  ) : null
+
+const MutedIcon = ({isLocked, isMuted}) =>
+  isMuted ? (
+    <>
+      <Icon className="muted_icon" type="icon-shh-16" style={avatarIconStyle} />
```

Comment:
i don't think we want to render 3 icons and just hide them, you can have one icon. maybe let me make an example since there isn't one currently

Reply:
Thanks!

Reply:
so i started doing this but its a lot of code and its likely not worth it, i think we should keep what we had before

Reply:
So, just leave it with the mouse handlers?

Reply:
yeah. in order for this to be easier i think how we handle css needs to be better

Reply:
Sounds good. Thanks for the feedback!